src/modules/header/styles.css

```diff
@@ -0,0 +1,268 @@
+@custom-media --header-breakpoint (min-width: 768px);
+
+:root {
+  --header-menu-width: 12rem;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  line-height: 1; /* FIXME: in another place */
+}
+
+.container li::before {
+  /**
+   * Fixes style breaking when sanitize.css inserts non-breaking space
+   * to prevent VoiceOver disrespecting list markup in <nav>
+   */
+  display: block;
+  height: 0;
+}
+
+.nav {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  padding: 0;
+  margin: 0 0 0 auto;
+  transition: .15s;
+
+  @media (--header-breakpoint) {
+    flex-direction: row;
+    width: auto;
+  }
+
+  &:not(.expanded) {
+    height: 0;
+    visibility: hidden;
+
+    @media (--header-breakpoint) {
+      height: auto;
+      visibility: visible;
+    }
+  }
+}
+
+
+.logo-container {
+  margin-right: auto;
+  font-weight: 600;
+}
+
+.logo {
+  height: 1rem;
+  margin-bottom: -1rem;
+}
+
+.toggle-container {
+  display: flex;
+  margin-left: auto;
+
+  @media (--header-breakpoint) {
+    display: none;
+  }
+}
+
+.toggle-button {
+  --icon-color: hsl(var(--component-color) / var(--component-medium-emphasis));
+  padding: .75rem;
+  margin: -1rem;
+  line-height: 1.5;
+  border-radius: 0;
+}
+
+.menu {
```

Comment:
I think the menu needs some z-index and background colour

![image](https://user-images.githubusercontent.com/12775440/99249653-bcf6b180-280a-11eb-9ae9-5bdce3e73108.png)

Reply:
There is undocumented `--header-zindex` (or 1000 by default) applied for the desktop only. I will move this for mobile too.

Reply:
I see the problem: if `position: fixed` is applied to the overall component, you lack the background and z-index. I am afraid, `z-index` should be applied manually.

Reply:
Also, you can wonderfully drop `fixed` prop there which gets automatically passed to the `AppBar` and solves your problem.

Reply:
I cannot drop position fixed because when you are on the mobile menu and open the submenu the main content gets moved down and I don't think it should work like that

Reply:
> There is undocumented `--header-zindex` (or 1000 by default) applied for the desktop only. I will move this for mobile too.

Renamed to `--header-z-index` and defaults to `--app-bar-z-index + 10`.

---

I don't know, perhaps not. But if you make the app-bar fixed, it won't do so, child menus will overlay the content of the page. I think it's good option.